resources/providers/aws/elb_provider.go

```diff
@@ -15,16 +15,21 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package fetchers
+package aws
```

Comment:
A general note: `aws` package name is confusing, maybe `awslib` instead?

Reply:
I struggled a lot with finding the best name for the package.

I just followed the beats convention at the end - the beats creates `aws` package for all the `aws` content.

But to be honest, I think that `awslib` is better, I hope that it won't confuse anyone.